src/network/NetworkObserver.ts

```diff
@@ -440,6 +434,20 @@ export class NetworkObserver implements Observer<NetworkRequest> {
     }
   }
 
+  private loadContent(networkRequest: NetworkRequest): void {
+    const allowedMimeType = Array.isArray(this.options.includeMimes)
+      ? this.options.includeMimes.includes(networkRequest.mimeType)
+      : false;
```

Comment:
Shouldn't this be `true`?

```suggestion
      : true;
```

Reply:
This is the reason why this PR remains in draft.